cmd/ferretdb/main.go

```diff
@@ -66,26 +66,48 @@ var cli struct {
 
 	Version bool `default:"false" help:"Print version to stdout and exit." env:"-"`
 
-	PostgreSQLURL string `name:"postgresql-url" default:"postgres://127.0.0.1:5432/postgres" help:"PostgreSQL URL." group:"PostgreSQL"`
+	PostgreSQLURL     string `name:"postgresql-url"      default:"postgres://127.0.0.1:5432/postgres"   help:"PostgreSQL URL." group:"PostgreSQL"`
+	PostgreSQLURLFile string `name:"postgresql-url-file" help:"Path to file containing PostgreSQL URL." group:"PostgreSQL"     type:"filecontent"`
```

Comment:
If `PostgreSQLURLFile` is set, `PostgreSQLURLFile` should be used to read url with secrets from this file. Could you implement that? Currently, `PostgreSQLURLFile` flag is not used.

Reply:
@chilagrow should i add test for above change?

Reply:
I don't think it would be easy, so no. But we need to update the documentation.